test/benchmark_http_client_test.cc

```diff
@@ -31,7 +31,7 @@ class BenchmarkClientHttpTest : public Test {
   BenchmarkClientHttpTest()
       : api_(Envoy::Api::createApiForTest()), dispatcher_(api_->allocateDispatcher()),
         cluster_manager_(std::make_unique<Envoy::Upstream::MockClusterManager>()),
-        cluster_info_(std::make_unique<Envoy::Upstream::MockClusterInfo>()),
+        cluster_info_(std::make_unique<NiceMock<Envoy::Upstream::MockClusterInfo>>()),
```

Comment:
I'm surprised this change is related.

Reply:
And rightly so, nice catch. Backed this out, the `NiceMock` wrapper is no longer needed.